src/components/Lights/Lights.component.tsx

```diff
@@ -0,0 +1,109 @@
+  keyLightPosition: new Vector3(0.5, 1.55, 0.5),
+  liftLightPosition: new Vector3(0.25, 1.7, 2.0),
+  defaultProps: {
+    keyLightIntensity: 1.2,
+    keyLightColor: '#e8e3df',
+    fillLightIntensity: 0.7,
+    fillLightColor: '#99ccff',
+    fillLightPosition: new Vector3(-0.5, 1.6, -0.5),
+    backLightIntensity: 2.3,
+    backLightColor: '#fff0d6',
+    backLightPosition: new Vector3(0.5, 1.6, -1.0),
+    lightTarget: new Vector3(0.0, 1.7, 0.0)
+  } as Required<LightingProps>
+});
+
+Object.fromEntries = (arr: Array<any>) => Object.assign({}, ...arr.map(([k, v]) => ({ [k]: v })));
+
+const definedProps = (obj: Object) =>
+  Object.fromEntries(
+    // eslint-disable-next-line
+    Object.entries(obj).filter(([k, v]) => v !== undefined)
+  );
+
+const Lights: FC<LightingProps> = (lightingProps) => {
+  // use default props as fallback if no custom lighting settings are provided
+  const {
+    keyLightIntensity,
+    keyLightColor,
+    fillLightIntensity,
+    fillLightColor,
+    fillLightPosition,
+    backLightIntensity,
+    backLightColor,
+    backLightPosition,
+    lightTarget
+  } = Object.assign(LIGHT_CONFIG.defaultProps, definedProps(lightingProps));
+
+  const { scene } = useThree();
+
+  const [targets] = useState<{ head: Object3D; shoe: Object3D }>({
+    head: new Object3D(),
+    shoe: new Object3D()
+  });
+
+  useEffect(() => {
+    // apply provided positions for targets
+    targets.head.position.copy(lightTarget);
+    targets.shoe.position.set(0.0, 0.0, 0.0);
+
+    // add targets to scene (without the spotlights would not aim at them)
+    scene.add(targets.head);
+    scene.add(targets.shoe);
+  }, []);
+
+  return (
+    <group>
+      {/* Fill light that by default creates strong blue rim on the right face side. */}
+      <spotLight
+        position={fillLightPosition}
+        target={targets.head}
+        angle={LIGHT_CONFIG.fillLightAngle}
+        color={fillLightColor}
+        intensity={fillLightIntensity}
+      />
+      {/* Back light that by default creates light warm rim on the left face side. */}
+      <spotLight
+        position={backLightPosition}
+        target={targets.head}
+        angle={LIGHT_CONFIG.backLightAngle}
+        color={backLightColor}
+        intensity={backLightIntensity}
+      />
+      {/* Key light that creates soft face light. */}
+      <spotLight
+        position={LIGHT_CONFIG.keyLightPosition}
+        target={targets.head}
+        angle={LIGHT_CONFIG.keyLightAngle}
+        color={keyLightColor}
+        intensity={keyLightIntensity}
+      />
+      {/* Lift light that creates soft light on body and shoes. */}
+      <spotLight
+        position={LIGHT_CONFIG.liftLightPosition}
+        target={targets.shoe}
+        angle={LIGHT_CONFIG.keyLightAngle}
+        color={keyLightColor}
+        intensity={keyLightIntensity * 0.16}
+      />
+      <ContactShadows opacity={1.0} scale={2.0} blur={1.0} far={0.25} resolution={256} color="#000000" />
```

Comment:
We should remove the other shadow-catcher.

Reply:
Maybe leaving the old shadow is more performant than the contact shadows. Especially having already optional AO in place which can achieve a similar look.

Reply:
Removed contact shadows for now.